src/Compilers/CSharp/Test/Semantic/Semantics/InheritanceBindingTests.cs

```diff
@@ -7653,5 +7639,149 @@ public class B : A
 }";
             CreateCompilation(source).VerifyDiagnostics();
         }
+
+        [Fact]
+        [WorkItem(31974, "https://github.com/dotnet/roslyn/issues/31974")]
+        public void Issue31974()
```

Comment:
Please add a note in docs/CSharp/Compiler Breaking Changes - VS2019.md

Reply:
> Please add a note in docs/compilers/CSharp/Compiler Breaking Changes - VS2019.md

I don't think this is a breaking change, this is a bug fix for producing an invalid assembly and for a regression from native compiler

---
In reply to: [245095253](https://github.com/dotnet/roslyn/pull/32041#discussion_r245095253) [](ancestors = 245095253)

Reply:
It is a regression from the native compiler, but everyone is on roslyn now, and it _is_ technically a breaking change.

---
In reply to: [245096054](https://github.com/dotnet/roslyn/pull/32041#discussion_r245096054) [](ancestors = 245096054,245095253)